### PR TITLE
feat: vote suggestions

### PR DESCRIPTION
**Pull request by ManuViola77**
## Links:

[Agregar una pagina de sugerencias/feedback para que la gente pueda proponer mejoras y votarlas](https://github.com/users/ManuViola77/projects/1/views/1#:~:text=Agregar%20una%20pagina%20de%20sugerencias/feedback%20para%20que%20la%20gente%20pueda%20proponer%20mejoras%20y%20votarlas)

## What & Why:

This PR adds the possibility to vote for other users suggestions. 

## Risks, Mitigation & Rollback:

- [ ] Safe to Revert *check this box if this PR can be reverted without incident*

### CODE REVIEW

File: src/constants/constants.js [hunk 1]
```diff
@@ -49,4 +49,5 @@ export const MAX_ATTEMPTS = 6;
 export const ADMITTED_WORDS_SIZES = [4, 5, 6];
 export const ACCEPTED_WORDS = ['MONDAY', 'TUESDAY', 'FRIDAY', 'SUNDAY', 'ROOTSTRAP'];
 
-export const VOTED_COLOR = 'action';
+export const NOT_VOTED_COLOR = 'white';
+export const VOTED_COLOR = '#ffae00';
```
Comment (jeisux92) [inline src/constants/constants.js:53]: do you have a file with a palette color for this values?

File: src/hooks/useSuggestions.js [hunk 2]
```diff
@@ -187,6 +210,48 @@ const useSuggestions = () => {
     return true;
   };
 
+  const filterCurrentUser = arrayData => arrayData.filter(({ id }) => id !== myId);
```
Comment (jeisux92) [inline src/hooks/useSuggestions.js]: i would change the name of the arrayData parameter to something more clear

File: src/hooks/useSuggestions.js [hunk 3]
```diff
@@ -187,6 +210,48 @@ const useSuggestions = () => {
...
+    }
+
+    try {
+      await updateDoc(doc(suggestionsRef, currentSuggestion.id), currentSuggestion);
+    } catch (err) {
+      // TODO: handle errors
+      console.error(err);
+    }
+
+    getSuggestions();
```
Comment (jeisux92) [inline src/hooks/useSuggestions.js]: add `await` since this is an async operation

File: src/constants/constants.js (hunk 1 above)
Comment (ManuViola77) [reply src/constants/constants.js:53]: right now no, I have wan for CSS but not for js. I want to fix this in a near future.

File: src/hooks/useSuggestions.js (hunk 2 above)
Comment (ManuViola77) [reply src/hooks/useSuggestions.js]: I can call it votes since I call this with negativeVotes and positiveVotes.

**Review by jeisux92 — APPROVED** [review]
Perfect

**Pull request merged**